cmd/daprd/main.go

```diff
@@ -350,6 +351,9 @@ func main() {
 			bindings_loader.NewOutput("azure.eventgrid", func() bindings.OutputBinding {
 				return eventgrid.NewAzureEventGrid(logContrib)
 			}),
+			bindings_loader.NewOutput("cron", func() bindings.OutputBinding {
```

Comment:
does this needs to be registered as input as well as cron is implementing both Read and Invoke.

Reply:
Input is already registered in the v0.9 release. This is just adding the output binding

Reply:
got it, LGTM overall, looks like there are some conflicts, could you please resolve them.